frontend/src/scripts/generate-cert.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+mkdir -p mkcert
+cd mkcert
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)